processors/processPlayerData.js

```diff
@@ -148,10 +148,12 @@ function processPlayerData({
   let totalKills = 0;
   let totalWins = 0;
   let totalCoins = 0;
+  let totalGamesPlayed = 0;
   Object.keys(statsObject).forEach((game) => {
     totalKills += statsObject[game].kills || 0;
     totalWins += statsObject[game].wins || 0;
     totalCoins += statsObject[game].coins || 0;
+    totalGamesPlayed += statsObject[game].games_played || 0;
```

Comment:
Other gamemodes are probably missing this

Reply:
missing that line or the availability of the games_played?
if you're talking about that line, I was more looking to add this into the GraphQL call only.
And if you're talking about the availability of the games_played, yeah there's more I can add I believe.

Reply:
Yes, talking about other games having `game_played`, it would be silly to call total games played when it's from only one game.